resources/lang/en/community.php

```diff
@@ -126,25 +126,4 @@
             'instructions' => 'click the heart button to proceed to the osu!store',
         ],
     ],
-    'slack' => [
```

Comment:
Sure, but the page is still available right now (https://new.ppy.sh/community/slack). So if we merge these translation changes, that page will be full of missing translation errors... 😛 

We'll delete the translations when we remove/replace the page.